doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -17,6 +17,9 @@
         * [3.2.1.7 Hardware Acceleration](#3217-hardware-acceleration)
         * [3.2.1.8 Scheduling Pods with Non-resilient Applications](#3218-scheduling-pods-with-non-resilient-applications)
     * [3.2.2 Container Networking Services](#322-container-networking-services)
+      * [3.2.2.1 Kubernetes Networking Semantics](#3.2.2.1)
+         * [3.2.2.1.1 Built in Kubernetes network functionality](#3.2.2.1.1)
+         * [3.2.2.1.2 Multi networking and Orchestration](#3.2.2.1.2)
```

Comment:
```suggestion
         * [3.2.2.1.2 Multiple Networks and Advanced Configurations](#3.2.2.1.2)
```
Maybe "advanced configurations" is not the best phrase, but I'd like to keep word "orchestration" for higher layer in the stack.